xml/System.Reflection.Metadata.Ecma335/FieldTypeEncoder.xml

```diff
@@ -83,7 +83,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Get <see cref="M:System.Reflection.Metadata.Ecma335.FieldTypeEncoder.CustomModifiers" /> to specify custom modifiers to the field signature.</summary>
         <returns>To be added.</returns>
```

Comment:
Looks like this is being flagged as needed by https://github.com/dotnet/runtime/issues/76049.

Reply:
Right, return value missing